codex_of_the_damned/templates/archetypes/index.html

```diff
@@ -681,4 +681,26 @@ <h2>Bibliodèque</h2>
 		</tbody>
 	</table>
 </div>
+
+{% trans trimmed
+
+%}
+<h2>Experimental Builds</h2>
+<p>
+	As the game develops, new archetypes appear and some disappear. This section
+	will be updated from time to time with decks that have been tried using a
+	significant change to common archetypes.
```

Comment:
I'd rather brand it as something open to contributions, what do you think ? Some kind of "decklists contributions" ? I'd rather stay as humble as possible here. Plus this could be a nice incentive for people to contribute ^_^